bears/c_languages/CPPLintBear.py

```diff
@@ -26,12 +30,20 @@ class CPPLintBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=79,
                          cpplint_ignore: typed_list(str)=(),
-                         cpplint_include: typed_list(str)=()):
+                         cpplint_include: typed_list(str)=(),
+                         indent_size: int=2,
+                         use_spaces: bool=2):
         """
         :param max_line_length: Maximum number of characters for a line.
         :param cpplint_ignore:  List of checkers to ignore.
         :param cpplint_include: List of checkers to explicitly enable.
+        :param use_spaces: Bool value whether to use spaces or not.
+        :param indent_size: Size of indentation when spaces are used.
         """
+        if not check_invalid_config(use_spaces, indent_size):
+            raise ValueError("CPPLint doesn\'t support indent_size != 2 or "
+ +                             "not using spaces.")
```

Comment:
really i don't see the point of having this setting if the user cannot customize it... what do you think @SarthakNijhawan ?